MSVC/gen-build-config.bat

```diff
@@ -49,79 +50,91 @@ SET $BUILDCONFIG_TEMPLATE=build_config.h.in
 SET $BUILDCONFIG_FILE=build_config.h
 SET $VERSIONINFO_FILE=MSVC\versioninfo.h
 
-SET $USE_SQLITE3 0
-SET $USE_POSTGRESQL 0
-SET $USE_LEVELDB 0
-SET $USE_REDIS 0
-SET $USE_ICONV 0
-SET $PACKAGING_FLAT 1
-SET $INSTALL_PREFIX ""
+SET $USE_SQLITE3=0
+SET $USE_POSTGRESQL=0
+SET $USE_LEVELDB=0
+SET $USE_REDIS=0
+SET $USE_ICONV=0
+SET $PACKAGING_FLAT=1
+SET $INSTALL_PREFIX=\\
+SET $PROJECTDIR=""
 
-SET $USE_ANY 0
-SET $VERBOSE 0
-SET $TEST 0
+
+SET $USE_ANY=0
+SET $VERBOSE=0
+SET $TEST=0
 
 :: Command-line argument parsing
 :PARSEARGS
-	IF /I "%~1" == "-h" (
-		CALL USAGE
+	IF /I "%~1" =="-h" (
+		CALL :USAGE
 		EXIT /B 0
-	) ELSE IF /I "%~1" == "--help" (
-		CALL USAGE
+	)
+	IF /I "%~1" =="--help" (
```

Comment:
As I visited my parents this weekend, who do have a windows computer, I took the opportunity to do a few tests.

ELSE IF *should* work. However, the entire IF - ELSE IF - ELSE IF - etc... - ELSE statement *must* contain valid code. A syntax error, or a command which is valid syntax only if an earlier condition is met will cause an error for the entire IF-ELSE sequence, even if the error is in the very last ELSE branch for example.

Of course, my initial attempt was quite buggy (thanks for fixing...), so that may have been the reason that it failed (?).

So I am changing it back to ELSE IF, so that invalid options can be detected as well, and an error message printed.